bazel_tools/scala_version.bzl

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+def _impl(ctx):
+    # Generates an empty BUILD file, because we do not need to build anything.
+    ctx.file(
+        "BUILD",
+        content = """exports_files(["index.bzl"])""",
+        executable = False,
+    )
+
+    version = ctx.os.environ.get("DAML_SCALA_VERSION", default = "2.12.12")
```

Comment:
Maybe we could use the [`--repo_env`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--repo_env) flag to make this available as a `--config` option. E.g. in `.bazelrc`
```
build:scala_2_13 --repo_env=DAML_SCALA_VERSION=2.13.3
fetch:scala_2_13 --repo_env=DAML_SCALA_VERSION=2.13.3
query:scala_2_13 --repo_env=DAML_SCALA_VERSION=2.13.3
sync:scala_2_13 --repo_env=DAML_SCALA_VERSION=2.13.3
```
and then `bazel build --config scala_2_13`.

Reply:
I like it, added and switched the CI job over